packages/webamp-modern-2/src/maki/interpreter.ts

```diff
@@ -294,25 +294,61 @@ class Interpreter {
           let argCount: number = klass.prototype[methodName].length;
 
           const methodDefinition = getMethod(guid, methodName);
-          assert(
-            argCount === (methodDefinition.parameters.length ?? 0),
-            `Arg count mismatch. Expected ${
-              methodDefinition.parameters.length ?? 0
-            } arguments, but found ${argCount} for ${klass.name}.${methodName}`
-          );
+          if (methodName.toLowerCase() != "init") {
+            assert(
+              argCount === (methodDefinition.parameters.length ?? 0),
+              `Arg count mismatch. Expected ${
+                methodDefinition.parameters.length ?? 0
+              } arguments, but found ${argCount} for ${
+                klass.name
+              }.${methodName}`
+            );
+          }
 
           const methodArgs = [];
           while (argCount--) {
             const a = this.stack.pop();
             methodArgs.push(a.value);
           }
           const obj = this.stack.pop();
+          if(!!!obj.value/* ==null */ 
+            //&& (methodName==='newitem' || methodName==='newattribute'|| methodName==='getparentlayout')
+            && ( 
+                (klass.name || '').toLowerCase() == 'winampconfig' || 
+                (klass.name || '').toLowerCase() == 'winampconfiggroup' || 
+                (klass.name || '').toLowerCase() == 'configclass' 
+                )
+            ){
```

Comment:
there is a call to 'winampConfigGroup' 's method,  that assuming it as a const, but we declare it as a class.
So, instead of error, I create a new instance to sove the method calling.
Do you have a better idea?

Reply:
What's the class/const the method returns? (and what's the method name?) (if you recall).

Reply:
I can only answer once we loading the WinampModern skin.
for now, I only working with MMD3 skin.

Reply:
Okay. I deleted that lines. we has no use-case. (no cases with MMD3)
I am sorry, it was premature anticipation.